spec/models/recipe_spec.rb

```diff
@@ -12,4 +27,13 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "Total cost of Ingredients" do
+    it "calculates the total cost of the recipes ingredients" do
+
+      expect(pasta_dish.total_cost).to eq(15)
+      expect(chicken_fingers.total_cost).to eq(11)
+      expect(mac_n_cheese.total_cost).to eq(0)
```

Comment:
Oh nice! I like that you're testing what happens if no ingredients have been added to a recipe yet.